### PR TITLE
Completing the `add class` snippet

### PR DESCRIPTION
**Pull request by krasimir**
There are two problems with the current suggestion of adding a new class under IE:

a) If there is no class assigned so far the new one is added with an interval infront. I.e. it looks `class=" my-class"` and not `class="my-class"`.
b) There is no check if the class is already added

Here is a jsbin with the new version: http://jsbin.com/lajuh/2/edit

### CODE REVIEW

**Comment by zackbloom** [comment]
Thanks for doing this, but unfortunately the place to make these changes is in the file structure of the comparisons pre-compilation: https://github.com/HubSpot/youmightnotneedjquery/tree/gh-pages/comparisons

**Comment by krasimir** [comment]
Thanks for the feedback. I'll make them there and will make a new PR.

**Comment by krasimir** [comment]
@zackbloom actually the file that edited is `comparisons/elements/add_class/ie8.js`. Isn't that the right file?

**Comment by zackbloom** [comment]
My mistake, thanks @krasimir

File: comparisons/elements/add_class/ie8.js [hunk 1]
```diff
@@ -1,4 +1,9 @@
-if (el.classList)
+if (el.classList) {
   el.classList.add(className);
-else
-  el.className += ' ' + className;
+} else {
+  var current = el.className;
+  if (current.indexOf(className) < 0) {
```
Comment (zackbloom) [inline comparisons/elements/add_class/ie8.js:5]: This is not a great test, you could have the class `error-message`, and then wouldn't be able to add the `error` class.

Comment (krasimir) [inline comparisons/elements/add_class/ie8.js:5]: You are completely right. Here is a new version that covers this use case https://github.com/HubSpot/youmightnotneedjquery/pull/75 Here is JSBin http://jsbin.com/ciqibori/3/edit?html,js,console

**Pull request closed without merging**